user-domain/src/main/java/life/qbic/authentication/domain/user/concept/UserId.java

```
@@ -44,7 +44,7 @@ public static UserId create() {
    * @return an instance of a new user id object
    * @since 1.0.0
    */
-  public static UserId from(String s) {
+  public static UserId from(String s) throws IllegalArgumentException {
     try {
       return new UserId(UUID.fromString(s));
     } catch (IllegalArgumentException ignored) {
```

the exception is ignored..should it be? and can it really throw one then?